docs/source/conf.py

```diff
@@ -350,6 +350,13 @@
     ("py:class", "scipy.sparse._csc.csc_matrix"),
     ("py:class", "pathlib.Path"),
     ("py:class", "pydantic.main.BaseModel"),
+    ("py:class", "langchain.callbacks.base.BaseCallbackHandler"),
+    ("py:class", "langchain.callbacks.base.BaseCallbackManager"),
+    ("py:class", "langchain.chains.base.Chain"),
+    ("py:class", "langchain.schema.memory.BaseMemory"),
+    ("py:class", "langchain.schema.retriever.BaseRetriever"),
+    ("py:class", "BaseRetriever"),
+    ("py:class", "mlflow.langchain.retriever_chain.RetrieverChain"),
```

Comment:
We probably don't need to document this class anymore, right? We can even prefix it with `_` so that sphinx doesn't try to generate docs for it